dashboards/mintel/haproxy/haproxy-app-performance.json

```diff
@@ -1638,7 +1638,7 @@
         "multi": true,
         "name": "code",
         "options": [],
-        "query": "label_values(haproxy_backend_http_responses_total{job=\"haproxy-exporter\"}, code)",
+        "query": "label_values(haproxy_backend_http_responses_total{job=\"ingress-controller/haproxy\"}, code)",
```

Comment:
is this right?
```
haproxy-exporter -> ingress-controller/haproxy 

```

Reply:
yes

Reply:
https://prometheus.svc.vault.gke-dev.mintelcloud.io/graph?g0.range_input=1h&g0.expr=haproxy_backend_http_responses_total%7Bjob%3D%22ingress-controller%2Fhaproxy%22%7D&g0.tab=1